src/components/DesktopNavbarMobile.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7294520)
```suggestion
import React from "react"
```